acl_setup.php

```diff
@@ -102,6 +107,8 @@
      // xl('Pharmacy Dispensary')
  $gacl->add_object('admin', 'ACL Administration'              , 'acl'      , 10, 0, 'ACO');
      // xl('ACL Administration')
+ $gacl->add_object('admin', 'Multipledb'                      ,'multipledb', 10, 0, 'ACO');
```

Comment:
As an aside, I'm not finding much documentation about the multipledb module. Anyone have more info?